openverse_catalog/dags/cleveland_museum_workflow.py

```diff
@@ -1,50 +1,23 @@
-from datetime import datetime, timedelta
-
-from airflow import DAG
-from airflow.operators.python import PythonOperator
+# airflow DAG (necessary for Airflow to find this file)
+import logging
+from datetime import datetime
 
+from util.dag_factory import create_provider_api_workflow
 from provider_api_scripts import cleveland_museum_of_art
-from util.operator_util import get_log_operator
-
 
-DAG_DEFAULT_ARGS = {
-    'owner': 'data-eng-admin',
-    'depends_on_past': False,
-    'start_date': datetime(2020, 1, 15),
-    'email_on_retry': False,
-    'retries': 3,
-    'retry_delay': timedelta(days=1),
-}
+logging.basicConfig(
+    format='%(asctime)s - %(name)s - %(levelname)s:  %(message)s',
+    level=logging.DEBUG)
```

Comment:
This seems to be repeated in every file. Can this be made into a common shared util?

Reply:
Yes, copying and pasting this, I actually thought it would be a good idea to create an abstract class for provider API scripts. Admittedly, this will make the code that was made as simple as possible, to attract new contributors, more complex (function to class jump). However, now we have a template script that can actually create the class, all that will be left to do is to write the missing functions. Maybe it wouldn't be such a great jump in complexity?

Reply:
Inheritance in classes does not seem like a very massive jump, even less so when most of the code is autogenerated. We should probably discuss this with the entire team but I like the inheritance model a lot!

Reply:
This sounds awesome 😄